### PR TITLE
feat(frontend): display stake of the validation nodes

### LINKED ISSUE #309: Show stake for validators
In the list of validators, it would be good to show the current stake of the validators.

Also, if the account confirmed (signed), which is true for validators - make it clickable for details.

### PR DESCRIPTION
**Pull request by icerove**
fix #309 and #305

### CODE REVIEW

File: frontend/src/libraries/explorer-wamp/nodes.ts [hunk 1]
```diff
@@ -39,22 +40,34 @@ export default class NodesApi extends ExplorerApi {
...
+        ]),
+        this.queryValidators(),
       ]);
+
+      nodes.map((node: NodeInfo) => {
+        validators.map((val: any) => {
+          if (val.account_id === node.accountId) {
+            node.stake = val.stake;
+          }
+        });
```
Comment (bowenwang1996) [inline frontend/src/libraries/explorer-wamp/nodes.ts:68]: this is too inefficient :(

Comment (icerove) [reply frontend/src/libraries/explorer-wamp/nodes.ts:68]: do you have other option to do so? @bowenwang1996

Comment (icerove) [reply frontend/src/libraries/explorer-wamp/nodes.ts:68]: will change to map

File: frontend/src/components/nodes/NodeRow.tsx [hunk 2]
```diff
@@ -32,7 +33,9 @@ export default class extends React.PureComponent<Props> {
         <Col md="7" xs="7">
           <Row>
             <Col className="node-row-title">
-              @{node.accountId}{" "}
+              @{node.accountId}
+              {"   "}
+              <span>${node.stake && <Balance amount={node.stake} />}</span>
```
Comment (frol) [inline frontend/src/components/nodes/NodeRow.tsx:38]: 1. I think we don't need that `$` sign
2. I would like to have some indication that it is a stake, not an account balance or anything else. @corwinharrell @kcole16 any suggestions on how to make this clear?

File: frontend/src/libraries/explorer-wamp/nodes.ts [hunk 3]
```diff
@@ -39,22 +40,34 @@ export default class NodesApi extends ExplorerApi {
...
+              ORDER BY node_id DESC
+              LIMIT :limit
+          `,
+          {
+            limit,
+            paginationIndexer,
+            validatorIndicator,
+          },
+        ]),
+        this.queryValidators(),
```
Comment (frol) [inline frontend/src/libraries/explorer-wamp/nodes.ts:60]: Let's decouple fetching the list of nodes based on telemetry from the RPC query. Nodes List page is quite important and we should not block the display when RPC is unreachable.

File: frontend/src/components/nodes/NodeRow.tsx (hunk 2 above)
Comment (kcole16) [reply frontend/src/components/nodes/NodeRow.tsx:38]: Something like "Staking 100,000 N"?

Comment (frol) [reply frontend/src/components/nodes/NodeRow.tsx:38]: @kcole16 keep it stupid simple! I love it! :smile:

File: frontend/src/components/nodes/NodeRow.tsx [hunk 4]
```diff
@@ -20,8 +25,28 @@ export default class extends React.PureComponent<Props> {
...
     ["NoSync", ""],
   ]);
+
+  state: State = { blockDiff: 0 };
+
+  componentDidMount() {
+    new BlocksApi().getLatestBlockHeight().then((lastBlockHeight) =>
+      this.setState({
+        blockDiff: Math.abs(
+          lastBlockHeight.lastBlockHeight - this.props.node.lastHeight
```
Comment (bowenwang1996) [inline frontend/src/components/nodes/NodeRow.tsx:35]: what is this for?

Comment (icerove) [reply frontend/src/components/nodes/NodeRow.tsx:35]: https://github.com/near/near-explorer/issues/305

Comment (frol) [reply frontend/src/components/nodes/NodeRow.tsx:35]: There is no point in saving it to state, it is actually more expensive to save it than to compute. Also, this latest block height should be fetched only once, and passed via React Context, so we don't query it for every single node. Also, make sure that it takes into account the auto-refresh logic.

**Comment by icerove** [comment]
Frol, I learned a lot about cypress test and find out it is really useful for end to end test. I recommend merge this PR for the feature of validators and create a new issue and PR for the cypress test since on local net, there is no node list on node page and on https://explorer.betanet.near.org/ there is no new feature listed. After this PR merge, I will write end-to-end test for most of explorer pages.
@frol

File: frontend/src/components/dashboard/DashboardHeader.tsx [hunk 5]
```diff
@@ -132,7 +132,10 @@ export default class extends React.Component<State> {
               text={
                 details ? (
                   <Link href="/nodes/[role]" as={`/nodes/validators`}>
-                    <a style={{ textDecoration: "none", color: "#24272a" }}>
+                    <a
+                      id="node-page"
```
[the remark this answers is not in the document]
Comment (icerove) [reply frontend/src/components/dashboard/DashboardHeader.tsx:136]: for simulate test

**Pull request merged**